samples/isilon_v270_k8s_121.yaml

```diff
@@ -140,13 +150,12 @@ spec:
 
       # tolerations: Define tolerations for the controller deployment, if required.
       # Default value: None
-      # Uncomment if nodes you wish to use have the node-role.kubernetes.io/master taint
       tolerations:
+      # Uncomment if nodes you wish to use have the node-role.kubernetes.io/master taint
```

Comment:
Please remove the examples related to master in tolerations. From 1.24 default example is only control-plane. Please remove for all the samples below

Reply:
So we have to remove it for `>= k8s 1.24`. It's a bit confusing as the above comment is for `k8s 1.21`.

Reply:
Done